file_store/src/lora_beacon_report.rs

```diff
@@ -22,6 +22,7 @@ pub struct LoraBeaconReport {
     pub tx_power: i32,
     pub timestamp: DateTime<Utc>,
     pub signature: Vec<u8>,
+    pub tmst: u32,
```

Comment:
i have a sep PR for the tmst inclusion https://github.com/helium/oracles/pull/151

But whichever gets in first....